tests/test_playbooks/fixtures/hostgroup-0.yml

```diff
@@ -6282,7 +6282,7 @@ interactions:
       Accept: [application/json;version=2]
       Cookie: [_session_id=368cd63b7e87604931a457c832b26c60]
     method: GET
-    uri: https://centos7-katello-3-12/api/operatingsystems?per_page=4294967296&search=name%3D%22TestOS%22&thin=True
+    uri: https://centos7-katello-3-12/api/operatingsystems?per_page=4294967296&search=title%3D%22TestOS%22&thin=True
```

Comment:
Oh, those files are _not_ meant to be edited by hand.
Have a look at:
https://github.com/theforeman/foreman-ansible-modules/blob/master/tests/README.md#guidedeline-to-writing-tests

Reply:
Do you have a quick guide on setting up test environment just for this? I am testing changes on real env, but I dont want to generate VCR yamls out of it

Reply:
https://github.com/theforeman/forklift can easily deploy vagrant VMs for you

Reply:
OK, this section of the main README might help:
https://github.com/theforeman/foreman-ansible-modules#how-to-test-modules-in-this-repository

I can really recommend using forklift here: `vagrant up centos7-katello-2.12`. Keep in mind, that the hostname would be `centos7-katello-2-12` with no dot.
In the best case, you would record this test against one of the foreman hosts, but i guess that is impossible until the operatingsystem module uses apypie.

Reply:
> In the best case, you would record this test against one of the foreman hosts, but i guess that is impossible until the operatingsystem module uses apypie.

Could you please elaborate on that? So I should test against katello, not foreman, otherwise it won't work, right?

Reply:
I think you meant katello 3.12, not 2.12, is that right?

Reply:
Yes. Both is correct. We have modules that don't work against plain Foreman, despite being officially for Foreman (the underlying library is Katello/Satellite centric). Ans yes 3.12 is the latest Katello release and what should be used.